static/js/ui_util.js

```diff
@@ -30,4 +30,34 @@ exports.blur_active_element = function () {
     document.activeElement.blur();
 };
 
+function update_lock_icon_for_stream(stream_name) {
+    const icon = $("#compose-lock-icon");
+    const streamfield = $("#stream_message_recipient_stream");
+    if (stream_data.get_invite_only(stream_name)) {
+        icon.show();
+        streamfield.addClass("lock-padding");
+    } else {
+        icon.hide();
+        streamfield.removeClass("lock-padding");
+    }
+}
+
+// In an attempt to decrease mixing, set stream bar
+// color look like the stream being used.
+// (In particular, if there's a color associated with it,
+//  have that color be reflected here too.)
+exports.decorate_stream_bar = function (stream_name, element, is_compose) {
+    if (stream_name === undefined) {
+        return false;
+    }
+    is_compose = is_compose || true;
```

Comment:
Is this `|| true` intentional?  Seems like it might be a leftover debugging state.

Reply:
Yeah. This shouldn't be here.